### DISCUSSION TITLE
怎么连这儿都有粉红啊

### DISCUSSION THREAD

**Discussion by DalianerLaoli** [The-Run-Philosophy-Organization/run#203] [General]
太晦气了，好不容易找到个干净的地方，粉蛆怎么无处不在无孔不入啊。
墙内那么多论坛平台都不能让你们找到存在感和归属感吗？
有没有什么屏蔽封禁之类的办法，真的是看到他们就感觉恶心。

**Comment by the-run-philosophy** [comment]
找不到了，毕竟“偷着乐”都被封了。
这些人在国内也无话可说了。

**Reply by ytgui** [reply]
现在“明着乐”都不可以了

**Comment by SunsetMkt** [comment]
无论如何，那些用户也不违反GitHub的使用规则（只有严重暴力、严重色情、对其它用户的骚扰是明确在社区守则里禁止的），不过您可以`Block user`（拉黑用户）。也许可以举报辱骂言论，但举报服务的处理速度非常感人。

**Reply by DalianerLaoli** [reply]
癞蛤蟆跳脚背，不咬人膈应人。
混了这么多年网，谁还明着骂人，达不到被举报的标准。
他们光是那些顺着五官往外喷鸡血的言论和表现就足以让我反胃了。

**Comment by leon-fudan** [comment]
我不是第一次看到举着自由言论标杆的人们，跳出来说要屏蔽其他人。这个错乱而可笑的世界。

**Reply by SunsetMkt** [reply]
拉黑是个很正常的操作，它就在用户页头像侧左下角，“Block or Report”，每个人都可以做，而且“屏蔽”这个操作并没有错。
言论自由不是靠每个用户不互相拉黑维持的，我们在一个社交平台上，而拉黑在社交中非常普遍。
言论自由不保证在说话之后是否会令人讨厌，这个道理很明显。事实上与其争吵，还不如拉黑，至少更清净。

**Reply by DalianerLaoli** [reply]
不用搁这预设前提批判大环境了，我可从来没说什么要言论自由，我就是单纯不想看到你们，拜拜了您呐。
哪儿有什么言论自由，墙内怎么被恶心的，墙外怎么恶心回来才是正道。

**Reply by luxi78** [reply]
屏蔽就是：你有说话的权力，我也有不听的权力。

当别人说些我不喜欢听的东西时，有捂住耳朵的权力，没毛病吧。
